src/routers/productExports.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from 'express';
+
+import createRouter from '@/utilities/functions/createRouter';
+import authorized from '@/middleware/authorized';
+import { UserType } from '@/models/User/UserType';
+import { StatusCode } from '@/models/statusCodes';
+
+const router = createRouter();
+
+router.get('/', async (req, res) => {
+  res.status(StatusCode.OK).send('Successfully retrieved all product exports') ;
+}, authorized(UserType.ADMIN));
+
+router.post('/', async (req, res) => {
+  res.status(StatusCode.CREATED).send('Successfully retrieved new product export');
+}, authorized(UserType.ADMIN));
+
+router.put('/:uuid', async (req, res) => {
+  res.status(StatusCode.OK).send('Successfully retrieved updated product export');
+}, authorized(UserType.ADMIN));
```

Comment:
These are just stubs right? We will figure out the authorized user types when we implement them?

Reply:
Yes